src/native/despacer/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(despacer VERSION "0.1.0" LANGUAGES C)
+
+# for consistency with windows
+set(CMAKE_STATIC_LIBRARY_PREFIX_C "")
+
+# despacer.lib
+add_library(despacer STATIC "./src/despacer.c")
+target_include_directories(despacer INTERFACE "./include")
+set_target_properties(despacer PROPERTIES POSITION_INDEPENDENT_CODE ON) #-fPIC
+
+add_compile_definitions(despacer INTERFACE __SSE4_1__=true)
+if(MSVC)
+    target_compile_options(despacer INTERFACE /arch:SSE4.1)
+else()
+    target_compile_options(despacer INTERFACE -msse4.1)
+endif()
```

Comment:
On my CPU although it supports many instruction sets, I get this error:

```ps1
FAILED: CMakeFiles/despacer.dir/src/despacer.c.obj
C:\PROGRA~1\LLVM\bin\clang.exe -DINTERFACE -D__SSE4_1__=true -Ddespacer  -g -Xclang -gcodeview -O0 -D_DEBUG -D_DLL -D_MT -Xclang --dependent-lib=msvcrtd -
MD -MT CMakeFiles/despacer.dir/src/despacer.c.obj -MF CMakeFiles\despacer.dir\src\despacer.c.obj.d -o CMakeFiles/despacer.dir/src/despacer.c.obj -c C:/minijson/src/native/despacer/src/despacer.c
C:/minijson/src/native/despacer/src/despacer.c:36:17: error: always_inline function '_mm_popcnt_u32' requires target featur
e 'popcnt', but would be inlined into function 'sse4_despace_branchless_u4' that is compiled without support for 'popcnt'
    pos += 16 - _mm_popcnt_u32(mask16);
                ^
```

![image](https://user-images.githubusercontent.com/16418197/125080904-feceb380-e08a-11eb-86dd-eb4ceaecebb7.png)